### ISSUE TITLE
[FEATURE] : Removing manually updated contributors to dynamically updated contributors

### ISSUE THREAD

**Issue by PritamSarbajna** [PritamSarbajna/tourism-website#98]
# **Is your feature request related to a problem? Please describe.**
The contributor section needs to be manually updated whenever any new contributor contributes to it. So it becomes problem and there is a issue always open for this.

# **Describe the solution you'd like**
Okay so we can remove the manual entry section and make use of `https://github.com/PritamSarbajna/tourism-website/graphs/contributors`, which can tell us about the contributors to the project.

# **Additional context**
We have to change the UI accordingly and check if the UI really suits for the website.

**Comment by amolgorithm** [comment]
@PritamSarbajna Yeah, this can be done relatively simply but the project would not be Pure JS. You would need Node.js to use REST API to request a fetch for Github repo data (e.g. contributors). Using Node.js will require some major adjustments to the code though, such as using an engine like EJS for a responsive front-end html page that is linked to the back-end server. In which case, you would need Node.js, node modules like Express and EJS. You would also need to replace the index.html page with an EJS file. So ultimately, the whole project needs to be converted to a Node.js project, and you require the REST API. This will require a lot of adjustments and changes, but it should be able to do, theoretically.

**Comment by PritamSarbajna** [comment]
@amolgorithm What you're saying is correct but for that we have to change the whole project to a node js project. But I want to make this project beginner friendly because there are a lot of folks who are contributing is relatively new in this field. So instead if you want to really fix it, I can make a API with FastApi and host it anywhere. I will provide you the endpoints so you can easily fetch the data. Is it okay with you ?...In that way we have to just make a fetch function in javascript and change a few lines of code in this project.

**Comment by amolgorithm** [comment]
@PritamSarbajna That sounds like a good idea! That is indeed okay with me. Github recommended REST, albeit Fast API should work more efficiently. That should work, we could try it out.

**Comment by PritamSarbajna** [comment]
Okay @amolgorithm, My sem exams are going on that's why I'm not active so much but give me few days once the exam finishes I will make the API and get back to you.

**Comment by amolgorithm** [comment]
Hey @PritamSarbajna, now that the learn more issue is done, we can move onto this. Let me know when you make the API and I can begin working on this issue.

**Comment by PritamSarbajna** [comment]
> Hey @PritamSarbajna, now that the learn more issue is done, we can move onto this. Let me know when you make the API and I can begin working on this issue.

Yeah I will let you know.
